src/ngMessages/messages.js

```diff
@@ -217,9 +280,9 @@ angular.module('ngMessages', [])
     *       <pre>myForm.myName.$error = {{ myForm.myName.$error | json }}</pre>
     *
     *       <div ng-messages="myForm.myName.$error" style="color:maroon">
-    *         <div ng-message="required">You did not enter a field</div>
-    *         <div ng-message="minlength">Your field is too short</div>
-    *         <div ng-message="maxlength">Your field is too long</div>
+    *         <div ng-message="'required'">You did not enter a field</div>
+    *         <div ng-message="'minlength'">Your field is too short</div>
+    *         <div ng-message="'maxlength'">Your field is too long</div>
```

Comment:
It was mentioned previously, but why is interpolation not good enough for this? Using interpolation would mean we could ship it in 1.3